modules/openxr/openxr_api.h

```diff
@@ -380,7 +380,7 @@ class OpenXRAPI {
 
 	RID action_set_create(const String p_name, const String p_localized_name, const int p_priority);
 	String action_set_get_name(RID p_action_set);
-	bool action_set_attach(RID p_action_set);
+	bool attach_action_sets(Vector<RID> p_action_sets);
```

Comment:
```suggestion
	bool attach_action_sets(const Vector<RID> &p_action_sets);
```
Unless you intend the vector to be modified